src/pages/msme/ViewCreditOptions.tsx

```diff
@@ -96,6 +97,7 @@ function ViewCreditOptions() {
         credit_product_id: option.id,
         borrower_size: values.borrower_size,
         sector: values.sector,
+        annual_revenue: values.annual_revenue || null,
```

Comment:
Why is `|| null` needed?

Reply:
Otherwise, this is sent as "annual_revenue: '' ", which is not accepted by the backend as a number is expected. To fix that, we set the field to null as now or we could remove the key  if a value is not set, something like https://github.com/react-hook-form/react-hook-form/issues/656#issuecomment-565029942

Reply:
All good - null is fine, I just didn't find any other cases in the codebase. This means `0` is changed to `null`, but I doubt anyone would want to intentionally enter a revenue of 0.

Reply:
> I just didn't find any other cases in the codebase. 

Yeap, because this one is our first optional field :)